### ISSUE TITLE
An error occurs when installing it as a  pre-commit hook

### ISSUE THREAD

**Issue by yipliu** [shellcheck-py/shellcheck-py#103]
Hello, I've been searching for a long time and still don't have an answer to the following error.

```shell
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/user/.cache/pre-commit/repo0hnmfp1m/py_env-python3/bin/python', '-mpip', 'install', '.')
return code: 1
stdout:
    Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple
    Processing /home/user/.cache/pre-commit/repo0hnmfp1m
      Preparing metadata (setup.py): started
      Preparing metadata (setup.py): finished with status 'done'
    Building wheels for collected packages: shellcheck-py
      Building wheel for shellcheck-py (setup.py): started
      Building wheel for shellcheck-py (setup.py): still running...
      Building wheel for shellcheck-py (setup.py): finished with status 'error'
      Running setup.py clean for shellcheck-py
    Failed to build shellcheck-py
stderr:
      error: subprocess-exited-with-error
      
      × python setup.py bdist_wheel did not run successfully.
      │ exit code: 1
      ╰─> [18 lines of output]
          /home/user/.cache/pre-commit/repo0hnmfp1m/py_env-python3/lib/python3.8/site-packages/setuptools/config/setupcfg.py:293: _DeprecatedConfig: Deprecated config in `setup.cfg`
          !!
          
                  ********************************************************************************
                  The license_file parameter is deprecated, use license_files instead.
          
                  By 2023-Oct-30, you need to update your project and remove deprecated calls
                  or your builds will no longer be supported.
          
                  See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
                  ********************************************************************************
          
          !!
            parsed = self.parsers.get(option_name, lambda x: x)(value)
          running bdist_wheel
          running build
          running fetch_binaries
          error: [Errno 104] Connection reset by peer
          [end of output]
      
      note: This error originates from a subprocess, and is likely not a problem with pip.
      ERROR: Failed building wheel for shellcheck-py
    ERROR: Could not build wheels for shellcheck-py, which is required to install pyproject.toml-based projects
Check the log at /home/user/.cache/pre-commit/pre-commit.log
```

The following is my `pre-commit-config.yaml` （To be honest, I'm not very familiar with pre-commit）
```YAML
  # shell scripts linter
  - repo: https://github.com/shellcheck-py/shellcheck-py
    rev: v0.9.0.2
    hooks:
      - id: shellcheck
 ```

**Comment by asottile** [comment]
`error: [Errno 104] Connection reset by peer` means there's some issue with your network -- probably a proxy or other system blocking your access to the shellcheck binaries
